frame/executive/src/lib.rs

```diff
@@ -1006,10 +1033,11 @@ mod tests {
 		new_test_ext(1).execute_with(|| {
 
 			Executive::initialize_block(&Header::new_from_number(1));
+			Executive::finalize_block();
 			// NOTE: might need updates over time if new weights are introduced.
 			// For now it only accounts for the base block execution weight and
 			// the `on_initialize` weight defined in the custom test module.
-			assert_eq!(<frame_system::Module<Runtime>>::block_weight().total(), 175 + 10);
+			assert_eq!(<frame_system::Module<Runtime>>::block_weight().total(), 175 + 175  + 10);
```

Comment:
seems like you should create some new tests for this new functionality

Reply:
hmmm so the test is named block_hooks_weight_is_stored and the note indicates that if new weights are introduced it should be added here (that is how I read it, but it would seem as if you wrote the note so if I mis-interpreted it no worries) . To me that made sense to put it in there, but if you think it shouldn't be I am happy to make a new test for it